catroid/src/test/java/org/catrobat/catroid/test/ui/regexassistant/JsonRegexExtractorTest.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2020 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.catroid.test.ui.regexassistant;
+
+import android.app.Activity;
+
+import org.catrobat.catroid.utils.JsonRegexExtractor;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static org.junit.Assert.assertEquals;
+
+public class JsonRegexExtractorTest {
+	@Before
+	public void setUp() {
+		Activity context = new Activity();
+		this.jsonExtractor = new JsonRegexExtractor(context);
```

Comment:
```suggestion
		this.jsonExtractor = new JsonRegexExtractor();
```
Context is not used. This is part of the fix to solve the static analysis warning "Avoid unused private fields such as 'context'."